packages/material-ui-lab/src/Pagination/usePagination.test.js

```diff
@@ -0,0 +1,128 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { assert } from 'chai';
+import { usePagination } from '.';
+
+describe('usePagination', () => {
+  it('has one page by default', () => {
+    const { items } = renderHook(() => usePagination()).result.current;
+    assert.strictEqual(items.length, 3);
```

Comment:
Going to drop some suggestions for better matchers here. The goal is to have a useful message once this test fails. 

The usual debugging procedure with this is that you 
1. see that the line failed
2. have to go to the source and find out what the issue is

By using a proper matcher you have these steps inside your console e.g.

```suggestion
    expect(items).to.have.length(3);
```
gives you `expected {} to have length of 3 but got 2` whereas before you would get `expected 2 to equal 3`. The second one doesn't help you at all. This isn't much of a problem if you write test after implementation (because it's basically just a regression test). But if another contributor changes seemingly unrelated code and CI fails then you want the hint to be explicit.